CONTRIBUTING.md

```diff
@@ -39,8 +39,46 @@ Or to be safe, you call apply all the required resources at once:
 
 1. `k-apply-all`
 
+### Releasing a New Version
+
+1. [Create Release Branch](#creating-a-release-branch) or [Patch a Release Branch](#patching-a-release-branch) the [Release Branch](#release-branches).
+1.
+Visit [Tag Workflow Action](https://github.com/crossplane-contrib/provider-upjet-digitalocean/actions/workflows/tag.yaml)
+and click on the `Run workflow` button. Make sure the "Branch" is set to the release branch, e.g., `release-0.2`.
+Enter the version number in the "Release version" field, e.g., `v0.2.1`, and a sensitive value for the "Tag message"
```

Comment:
curious, but what does "sensitive" mean here? typically I see that word used for secret information, but I don't think that's what is meant here. Perhaps "sensible" is the intended word?